packages/admin-next/dashboard/src/routes/products/product-create/components/product-create-variants-form/product-create-variants-form.tsx

```diff
@@ -167,10 +174,21 @@ const useColumns = ({
       }),
 
       ...currencies.map((currency) => {
+        const preference = pricePreferences.find(
+          (p) => p.attribute === "currency_code" && p.value === currency
+        )
+
         return columnHelper.column({
           id: `price_${currency}`,
           name: `Price ${currency.toUpperCase()}`,
-          header: `Price ${currency.toUpperCase()}`,
+          header: () => (
+            <div className="flex w-full items-center justify-between gap-3">
+              {t("fields.priceTemplate", {
+                regionOrCountry: currency,
```

Comment:
nit(non-blocking): Is this supposed to be `regionOrCountry`? Seems off given we are passing a `currency`

Reply:
No it's wrong, I forgot to update it. I'll change it to currency